striketracker/config.go

```diff
@@ -21,25 +21,33 @@ func (c *Configuration) Validate() error {
 	return nil
 }
 
-// Config is a functional API for configuring the client
-type Config func(*Configuration)
+// Option is a functional API for configuring the client.
+// https://commandcenter.blogspot.com/2014/01/self-referential-functions-and-design.html
+type Option func(*Configuration)
```

Comment:
This is fine, I'm still in that phase of development where things are getting renamed a lot and this is definitely  more appropriate